pkg/build/nodeimage/const_storage.go

```diff
@@ -20,43 +20,49 @@ package nodeimage
 The default PV driver manifest and images are provisionally rancher.io/local-path-provisioner
 NOTE: we have customized it in the following ways:
 - storage is under /var instead of /opt
-- debian-base is used as the helper image (k8s already ships this upstream as the base for many images) instead of busybox
-- schedule to linux nodes
+- our own image and helper image
+- schedule to linux nodes only
 - install as the default storage class
 */
 
-var defaultStorageImages = []string{"docker.io/rancher/local-path-provisioner:v0.0.14", "k8s.gcr.io/build-image/debian-base:buster-v1.7.2"}
+const storageProvisionerImage = "docker.io/kindest/local-path-provisioner:v0.0.22-kind.0"
+const storageHelperImage = "docker.io/kindest/local-path-helper:v20220512-507ff70b"
+
+// image we need to preload
+var defaultStorageImages = []string{storageProvisionerImage, storageHelperImage}
 
 const defaultStorageManifest = `
```

Comment:
this is the v0.0.22 deploy.yaml, but then:
- change the storage class to be default/standard
- swap out the image and helper image
- set the default storage location

same as we did before, but I've normalized the images to a single source of truth